tests/frontend/test_decompositions.py

```diff
@@ -100,18 +100,18 @@ def test_symmetric_validation(self):
         with pytest.raises(ValueError, match="matrix is not symmetric"):
             dec.graph_embed(A)
 
-    def test_mean_photon(self, tol):
+    def test_max_mean_photon_deprecated(self, tol):
```

Comment:
It's probably find as is, but would it make sense to have two separate classes `TestGraphEmbed` and `TestGraphEmbedDeprecated`, as currently the `test_square_validation` and `test_symmetric_validation` tests no longer apply to `graph_embed_deprecated`?

Reply:
I parametrized the test so that it uses both graph_embed and graph_embed_deprecated. Not a big fan of classes.